timemachine/fe/single_topology_v3.py

```diff
@@ -779,34 +995,71 @@ def setup_intermediate_state(self, lamb):
         """
         src_system = self.src_system
         dst_system = self.dst_system
-        interpolate_fn = interpolate.linear_interpolation
 
-        # tbd: use different interpolation functions later
+        # alchemical parameter values at which angle and torsion force constants transition from zero to nonzero for
+        # ring closing transformations
+        lambda_angles = 0.4  # for angles, src_k = 0 => k(lamb) = 0 when lamb < lambda_angles
```

Comment:
Added the ability to pass overrides for the parameter interpolation functions to `setup_intermediate_state` in [3003770](https://github.com/proteneer/timemachine/pull/820/commits/300377099ccf6981338d5c196bc508fc3289f815) (this will also be useful for setting up controls in stability tests)